### DISCUSSION TITLE
Getting blank page when trying to authenticate

### DISCUSSION THREAD

**Discussion by kartu** [abraunegg/onedrive#1340] [Q&A]
### Bug Report Details ###
Following the link printed by onedrive executable, after being prompted for password and successfully authenticating, I'm getting a blank page with the following URL:

https://login.microsoftonline.com/common/oauth2/nativeclient?code=......

**Application and Operating System Details:**
Linux Mint 20.1 CInnamon  
Kernel 5.4.0-66-generic
*   Are you using a headless system (no gui) or with a gui installed?
GUI
*   OneDrive Account Type
Personal

*   Did you build from source or install from a package?
Built from the latest source (git hash 424454b29214b85fab8156046c2e5936739a557e )

*   If you installed from source, what is your DMD or LDC compiler version: `dmd --version` or `ldmd2 --version`
DMD64 D Compiler v2.090.1

*   OneDrive Application Version: Output of `onedrive --version`
onedrive version                       = v2.4.10-3-g424454b

*   Provide the version of curl you are using: Output of `curl --version`
curl 7.68.0 (x86_64-pc-linux-gnu) libcurl/7.68.0 OpenSSL/1.1.1f zlib/1.2.11 brotli/1.0.7 libidn2/2.2.0 libpsl/0.21.0 (+libidn2/2.2.0) libssh/0.9.3/openssl/zlib nghttp2/1.40.0 librtmp/2.3

**Comment by abraunegg** [comment]
@kartu 
Please re-read the usage document on authorizing the client: https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#authorize-the-application-with-your-onedrive-account
